src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/src/RazorRenameFeature.ts

```diff
@@ -0,0 +1,96 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+import * as vscode from 'vscode';
+import { getRazorDocumentUri, isRazorCSharpFile } from './RazorConventions';
+import { RazorDocumentManager } from './RazorDocumentManager';
+import { RazorDocumentSynchronizer } from './RazorDocumentSynchronizer';
+import { RazorLanguage } from './RazorLanguage';
+import { RazorLanguageServiceClient } from './RazorLanguageServiceClient';
+import { RazorLogger } from './RazorLogger';
+import { RazorRenameProvider } from './RazorRenameProvider';
+import { LanguageKind } from './RPC/LanguageKind';
+
+export class RazorRenameFeature {
+    public readonly serviceClient: RazorLanguageServiceClient;
```

Comment:
Don't need to be public. Also can remove both of these in favor of the `private` parameters in the ctor